e2e/test/Helpers/logging/TestLogger.cs

```diff
@@ -51,6 +51,8 @@ private TestLogger()
                     { LoggingPropertyNames.TestRunId, Guid.NewGuid().ToString() },
                     // Build Id in the pipeline.
                     { LoggingPropertyNames.BuildId, Environment.GetEnvironmentVariable("BUILD_BUILDID") },
+                    { LoggingPropertyNames.Attempt, Environment.GetEnvironmentVariable("SYSTEM_JOBATTEMPT") },
```

Comment:
Adding this data to help get better statistics on which tests are failing on PR builds and CI builds.